### ISSUE TITLE
Will this work in Joomla 5?

### ISSUE THREAD

**Issue by CharlieBigley** [ceford/j4xdemos-plg-onoffbydate#2]
Hi Cliff, I'd be grateful if you could confirm that this code should install and work for Joomla 5

**Comment by ceford** [comment]
Yes! At least, the zip file I created in October 2021 installs and works in J5. That was one of my early attempts at coding for documentation. I guess it doesn't meet current coding standards. I am currently looking at my old tutorials with a view to revising them and the code examples. Comments welcome!
